osu.Game/Storyboards/Drawables/DrawableStoryboardSprite.cs

```diff
@@ -79,15 +79,17 @@ public override Anchor Origin
             {
                 var origin = base.Origin;
 
-                if (FlipH)
+                // Either flip horizontally or negative X scale, but not both.
+                if (FlipH ^ (VectorScale.X < 0))
                 {
                     if (origin.HasFlagFast(Anchor.x0))
```

Comment:
Can these not be updated to use the helper method?

Reply:
Whoops...